Project.toml

```diff
@@ -31,7 +32,7 @@ Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 FlashRankPromptingToolsExt = ["FlashRank"]
 GoogleGenAIPromptingToolsExt = ["GoogleGenAI"]
 MarkdownPromptingToolsExt = ["Markdown"]
-RAGToolsExperimentalExt = ["SparseArrays", "LinearAlgebra", "Unicode"]
+RAGToolsExperimentalExt = ["SparseArrays", "LinearAlgebra", "Unicode", "Pinecone"]
```

Comment:
Pinecone will have to a separate extension in the need. Otherwise everyone would need it even for in memory indexes